pkgs/apache/maven/pkg.yaml

```diff
@@ -1,2 +1,41 @@
 packages:
+  - name: apache/maven@maven-3.0.4
+  - name: apache/maven@maven-3.0.5
+  - name: apache/maven@maven-3.1.0
+  - name: apache/maven@maven-3.1.1
+  - name: apache/maven@maven-3.2.1
+  - name: apache/maven@maven-3.2.2
+  - name: apache/maven@maven-3.2.3
+  - name: apache/maven@maven-3.2.5
+  - name: apache/maven@maven-3.3.1
+  - name: apache/maven@maven-3.3.3
+  - name: apache/maven@maven-3.3.9
+  - name: apache/maven@maven-3.5.0
+  - name: apache/maven@maven-3.5.2
+  - name: apache/maven@maven-3.5.3
+  - name: apache/maven@maven-3.5.4
+  - name: apache/maven@maven-3.6.0
+  - name: apache/maven@maven-3.6.1
+  - name: apache/maven@maven-3.6.2
+  - name: apache/maven@maven-3.6.3
+  - name: apache/maven@maven-3.8.1
+  - name: apache/maven@maven-3.8.2
+  - name: apache/maven@maven-3.8.3
+  - name: apache/maven@maven-3.8.4
+  - name: apache/maven@maven-3.8.5
+  - name: apache/maven@maven-3.8.6
+  - name: apache/maven@maven-3.8.7
+  - name: apache/maven@maven-3.8.8
+  - name: apache/maven@maven-3.8.9
+  - name: apache/maven@maven-3.9.0
+  - name: apache/maven@maven-3.9.1
+  - name: apache/maven@maven-3.9.2
+  - name: apache/maven@maven-3.9.3
+  - name: apache/maven@maven-3.9.4
+  - name: apache/maven@maven-3.9.5
+  - name: apache/maven@maven-3.9.6
+  - name: apache/maven@maven-3.9.7
+  - name: apache/maven@maven-3.9.8
+  - name: apache/maven@maven-3.9.9
+  - name: apache/maven@maven-3.9.10
```

Comment:
Note that this is just test data.
You can install versions not listed here.
The test data is too many.
I'll reduce this.